schematic/manifest/generator.py

```diff
@@ -1236,7 +1242,7 @@ def _gather_all_fields(self, fields, json_schema):
         )
         return required_metadata_fields
 
-    def get_empty_manifest(self, json_schema_filepath=None):
+    def get_empty_manifest(self, json_schema_filepath=None, sheet_url=None):
```

Comment:
since that we will deprecate `sheet_url` in the future, is there a way to get away from adding `sheet_url` parameters to more places? What about using `output_format` parameter?

Reply:
For now this is the only direct way of specifying that we are using google sheets. I imagine `sheet_url` will be depreciated when you refactor the generator, and in that case I think the rewrite will be significant enough that specifying `sheet_url` here shouldn't be too much of a burden.